src/main/listeners.ts

```diff
@@ -13,9 +13,9 @@ import { fetchNewsFeedData } from "./newsFeed";
 import { verifyIso } from "./verifyIso";
 
 export function setupListeners() {
-  ipcMain.on("onDragStart", (event, filePath: string) => {
+  ipcMain.on("onDragStart", (event, filePaths: string[]) => {
     event.sender.startDrag({
-      file: filePath,
+      files: filePaths, // if this shows an error, its due to the development api being outdated
```

Comment:
```suggestion
      file: filePaths,
```

the API accepts `string[]` for `file`.

Reply:
Yes, but I'm getting a runtime error when passing in a string[] into file, although the api recommends that file accepts `string | string[]`.

Using files instead worked out, but typescript marks an error there.